app/views/landing_page/blocks/_side_navigation.html.erb

```diff
@@ -1,6 +1,11 @@
+<%
+  add_view_stylesheet("landing_page/side-navigation")
+%>
+<div class="side-nav">
 <%= render "govuk_publishing_components/components/contents_list", {
```

Comment:
This component has the component wrapper on it, so you should just be able to pass `classes: "side-nav"` through to the component, instead of adding another div element. 

https://components.publishing.service.gov.uk/component-guide/contents_list/#standard-options

Reply:
Ah yeah I forgot about that - ignore me!